config.sample.toml

```diff
@@ -1,23 +1,28 @@
-# Uncomment to enable GPT rewrite
+# Uncomment to enable GPT rewrite by OpenAI (exclude Azure)
 [openai]
+# Find the params of openai.OpenAI(...) from https://github.com/openai/openai-python
 
 # Set up your OpenAI API key, otherwise it will be read from the OPENAI_API_KEY environment variable
 # api_key = ""
 
 # Set up your OpenAI API base, otherwise it will be read from the OPENAI_API_BASE environment variable or default to https://api.openai.com/v1
-# api_base = "https://api.openai.com/v1"
+# base_url = "https://api.openai.com/v1"
 
-# Set to "azure" to use the Azure API, otherwise it will use the default OpenAI API
-# api_type = "open_ai"
+#=====================================
+# Uncomment to enable GPT rewrite by Azure OpenAI
+# [azure_opeai]
+# Find the params of openai.AzureOpenAI(...) from https://github.com/openai/openai-python
 
-# Set http proxy url like http://localhost:8080 to connect to the API through a proxy
-# proxy = ""
```

Comment:
openai now use httpx to make requests, so the proxy setting is not needed to be specifictly configured now.

Reply:
found it ....

Reply:
and httpx win the game, https://github.com/encode amazing